lib/keycloak.go

```diff
@@ -112,3 +323,15 @@ func decodeSecretValue(data map[string][]byte, key string) (ctrl.Result, string,
 	}
 	return ctrl.Result{}, string(decStr), nil
 }
+
+func getKeycloakGroupName(tenantName string) string {
+	return tenantName + "-tenant-admin"
+}
+
+func getKeycloakBase() string {
+	return "http://keycloak.services:8080/keycloak"
```

Comment:
Is this something that could be fetched instead of hard coded?  also, this looks like it is hitting the internal dns name but you are hitting the external endpoint instead of 

NAME                 TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)           AGE
cray-keycloak-http   ClusterIP   10.22.246.22   <none>        80/TCP,8443/TCP   46m